specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/preview/2018-06-30-preview/examples/getSoftwareUpdateProfile.json

```diff
@@ -0,0 +1,50 @@
+{
+  "parameters": {
+    "subscriptionId": "mySubscriptionId",
+    "resourceGroupName": "myResourceGroupName",
+    "vmName": "myVMName",
+    "api-version": "2018-06-30-preview"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "id": "/subscriptions/mySubscriptionId/resourceGroups/myResourceGroupName/providers/Microsoft.Compute/virtualMachines/myVMName/providers/Microsoft.GuestConfiguration/softwareUpdateProfile/default",
+        "name": "default",
+        "type": "Microsoft.GuestConfiguration/softwareUpdateProfile",
+        "properties": {
+          "computerName": "myVMName",
+          "computerIP": "40.85.217.32",
+          "computerEnvironment": "Azure",
+          "osType": "Windows",
+          "osName": "Microsoft Windows 10 Enterprise",
+          "osVersion": "10.0.17763 N/A Build 17763",
+          "osMajorVersion": "",
+          "osMinorVersion": "",
+          "osFullName": "",
+          "osArch": "x64",
+          "computerStartTime": "2018-10-30T17:31:02Z",
+          "restartPending": "True",
+          "restartPendingSince": "2018-11-01T12:34:00Z",
+          "guestConfigAgentVersion": "8.0.11081.0",
+          "guestConfigAgentStartTime": "2018-10-30T17:31:02Z",
+          "netRuntimeVersion": ".NET Framework 4.6.1",
+          "windowsUpdateSetting": "Notify before installation",
+          "windowsUpdateAgentVersion": "10.0.14393.2368",
+          "windowsUpdateIncludeRecommendedUpdates": "",
+          "wsusServer": "http://rd2-sup-hlb.redmond.corp.microsoft.com:80",
+          "linuxPackageManager": "",
+          "criticalCount": "2",
+          "securityCount": "0",
+          "otherCount": "0",
+          "approvalSource": "",
+          "latestAssessmentTimeTaken": "2018-10-30T17:31:02Z",
```

Comment:
This also changed. See https://msazure.visualstudio.com/One/_git/mgmt-automation-agentdataservice?path=%2Fsrc%2FSamples%2FPayload%2FSoftwareUpdatesPayload.json&version=GBmaster

Reply:
Same for SoftwareUpdateProfile. GuestConfig RP is the source of truth.
https://msazure.visualstudio.com/One/_git/Mgmt-Automation-GuestConfiguration?path=%2Fsrc%2FGuestConfiguration.ViewModel%2FSoftwareUpdateProfileV1.cs&version=GBmaster

Reply:
[Task](https://msazure.visualstudio.com/One/_workitems/edit/3696270) applies here too.